compiler/Makefile

```diff
@@ -10,54 +10,40 @@ libdir   ?= $(exec_prefix)/lib
 .PHONY: default
 default: stage$(STAGE)
 
-.PHONY: runtime/libskip_runtime64.a
-runtime/libskip_runtime64.a:
-	$(MAKE) -C runtime $(@F)
-
-.PHONY: runtime/libbacktrace/.libs/libbacktrace.a
-runtime/libbacktrace/.libs/libbacktrace.a:
-	$(MAKE) -C runtime libbacktrace
-
-.PHONY: runtime/libskip_runtime32.bc
-runtime/libskip_runtime32.bc:
-	$(MAKE) -C runtime $(@F)
-
-bootstrap/%.ll: bootstrap/%.ll.gz
-	gunzip -c $< > $@
-
 stage%:
 	mkdir -p $@
-	$(MAKE) $@/bin/skc $@/bin/skfmt $@/bin/skargo $@/lib/libskip_runtime64.a $@/lib/libbacktrace.a $@/lib/libskip_runtime32.bc $@/lib/skip_preamble64.ll $@/lib/skip_preamble32.ll
+	$(MAKE) $@/bin/skc $@/bin/skfmt $@/bin/skargo $@/lib/libstd.sklib $@/lib/libstd-wasm32-unknown-unknown.sklib
 
-stage%/lib/libskip_runtime64.a: runtime/libskip_runtime64.a
+stage%/lib/libstd.sklib: bootstrap/libstd.sklib
 	mkdir -p $(@D)
 	cp $< $@
 
-stage%/lib/libbacktrace.a: runtime/libbacktrace/.libs/libbacktrace.a
+# TODO: Get rid of this once skargo handles building sklibs.
+stage%/lib/libstd-wasm32-unknown-unknown.sklib: bootstrap/libstd-wasm32-unknown-unknown.sklib
 	mkdir -p $(@D)
 	cp $< $@
 
-stage%/lib/libskip_runtime32.bc: runtime/libskip_runtime32.bc
-	mkdir -p $(@D)
-	cp $< $@
+bootstrap/%.ll: bootstrap/%.ll.gz
+	gunzip -c $< > $@
 
-stage%/lib/skip_preamble64.ll: preamble/preamble64.ll
-	mkdir -p $(@D)
-	cp $< $@
+.PHONY: bootstrap/libskip_runtime64.a bootstrap/libbacktrace.a
+bootstrap/libskip_runtime64.a:
+bootstrap/libbacktrace.a:
+	OUT_DIR=$(realpath ./bootstrap) $(MAKE) -C ../prelude
```

Comment:
I may be getting confused between this and the skargo 2 PR, but this looks like it is putting host native object code in the bootstrap directory. Am I not understanding, or aren't we going to need the bootstrap dir to contain both x86_64 and arm64 native, as well as wasm, versions of the runtime?